lib/executor/constant_arrival_rate_test.go

```diff
@@ -110,7 +109,9 @@ func TestConstantArrivalRateRunCorrectRate(t *testing.T) {
 
 //nolint:tparallel,paralleltest // this is flaky if ran with other tests
 func TestConstantArrivalRateRunCorrectTiming(t *testing.T) {
-	// t.Parallel()
+	if runtime.GOOS == "windows" {
+		t.Skip()
```

Comment:
```suggestion
		t.Skip("This test is flaky on windows platform")
```

Reply:
Resolved by [this](https://github.com/grafana/k6/compare/4a61d014fb134ea05b6ac4f98e92b0c88f2505a9..02fb0b8c1f092c5f3a9d2e492806005da560fa7d) force-push